xhprof/docker-compose.override.yml

```diff
@@ -0,0 +1,137 @@
+version: '3.7'
+
+services:
+
+  ##
+  # The web server container.
+  ##
+  wordpress-develop:
+    image: nginx:alpine
+
+    networks:
+      - wpdevnet
+
+    ports:
+      - ${LOCAL_PORT-8889}:80
+
+    environment:
+      LOCAL_DIR: ${LOCAL_DIR-src}
+
+    volumes:
+      - ./tools/local-env/default.template:/etc/nginx/conf.d/default.template
+      - ./:/var/www
+
+    # Load our config file, substituting environment variables into the config.
+    command: /bin/sh -c "envsubst '$$LOCAL_DIR' < /etc/nginx/conf.d/default.template > /etc/nginx/conf.d/default.conf && exec nginx -g 'daemon off;'"
+
+    depends_on:
+      - php
+
+  ##
+  # The PHP container.
+  ##
+  php:
+    image: spacedmonkey/php-xhprof:${LOCAL_PHP-latest}
+    networks:
+      - wpdevnet
+
+    environment:
+      - LOCAL_PHP_XDEBUG=${LOCAL_PHP_XDEBUG-false}
```

Comment:
It would be great to support a environment variable like `LOCAL_PHP_XHPROF` similar to this one, so that it's easy to turn this on and off.

Reply:
You can turn it off with WP_CACHE like this. Once this is in core, this is how I want this to work.

Reply:
@spacedmonkey Thanks, that makes sense to me 👍